### PR TITLE
all: no multi account batches

### PR DESCRIPTION
**Pull request by manelmontilla**
This PR modifies the code so now it scans accounts one by one instead of scanning them concurrently in batches.
It also refactors the code related to postprocessing the graph generated by Altimeter to make it simpler.

### CODE REVIEW

File: graph_altimeter/scan/config.py [hunk 1]
```diff
@@ -92,20 +92,20 @@ def from_env(cls):
...
         self,
-        target_accounts_ids,
-        target_accounts_role,
+        target_account_id,
+        target_account_role,
         trampoline_account_role_arn=None,
     ):
-        """This function returns a dictionary suitable run an Altimeter Scan by
+        """This function returns a dictionary suitable to run an Altimeter Scan by
         using the base dictionary stored in the instance, and the given
```
Comment (jroimartin) [inline graph_altimeter/scan/config.py]: s/and// ?

File: graph_altimeter/scan/config.py [hunk 2]
```diff
@@ -92,20 +92,20 @@ def from_env(cls):
...
-        ``accounts_ids`` to scan, the ``target_accounts_role`` to be assumed in
-        those accounts, and a optional trampoline account."""
+        ``accounts_id`` to scan, the ``target_account_role`` to be assumed in
```
Comment (jroimartin) [inline graph_altimeter/scan/config.py]: s/accounts_id/target_account_id/ ?

File: graph_altimeter/scan/config.py [hunk 3]
```diff
@@ -92,20 +92,20 @@ def from_env(cls):
...
+        that account, and a optional trampoline account."""
```
Comment (jroimartin) [inline graph_altimeter/scan/config.py]: s/, and/and/ ?

**Review by jroimartin — CHANGES_REQUESTED** [review]
LGTM. I just added a couple a comments for typos.

File: bin/graph_altimeter_batch.py [hunk 4]
```diff
@@ -49,26 +49,12 @@ def run_scan():
         accounts = accounts_to_scan.split(",")
     else:
         accounts = get_aws_accounts(asset_inventory_api_url)
-    accounts_per_batch = os.getenv("ACCOUNTS_BATCH", None)
```
Comment (jroimartin) [inline bin/graph_altimeter_batch.py:52]: We should also remove `ACCOUNTS_BATCH` from [/env/local.env](https://github.com/adevinta/graph-altimeter/blob/7cf9e567efc12856a958b92cd554d240a5c5a679/env/local.env#L11-L12)

**Review by jroimartin — CHANGES_REQUESTED** [review]

File: graph_altimeter/scan/config.py (hunk 1 above)
Comment (manelmontilla) [reply graph_altimeter/scan/config.py]: yes right.

File: graph_altimeter/scan/config.py (hunk 2 above)
Comment (manelmontilla) [reply graph_altimeter/scan/config.py]: Also correct thanks.

File: graph_altimeter/scan/config.py (hunk 3 above)
Comment (manelmontilla) [reply graph_altimeter/scan/config.py]: 👍

File: bin/graph_altimeter_batch.py (hunk 4 above)
Comment (manelmontilla) [reply bin/graph_altimeter_batch.py:52]: good catch thank you.

**Review by jroimartin — APPROVED** [review]
LGTM

**Pull request merged**